tests/acceptance/features/bootstrap/TestingAppContext.php

```diff
@@ -870,6 +875,201 @@ public function theAdministratorHasCreatedFollowingLocks(TableNode $table) {
 		}
 	}
 
+	/**
+	 * @When the administrator adds url :url as trusted server using the testing API
+	 *
+	 * @param string $url
+	 *
+	 * @return void
+	 */
+	public function theAdministratorAddsUrlAsTrustedServerUsingTheTestingApi($url) {
+		$adminUser = $this->featureContext->getAdminUsername();
+		OcsApiHelper::sendRequest(
+			$this->featureContext->getBaseUrl(),
+			$adminUser,
+			$this->featureContext->getAdminPassword(),
+			'POST',
+			$this->getBaseUrl("/trustedservers"),
+			['url' => $url]
+		);
+	}
+
+	/**
+	 * @Then url :url should be a trusted server
+	 *
+	 * @param string $url
+	 *
+	 * @return  void
+	 */
+	public function urlShouldBeATrustedServer($url) {
+		$trustedServers = $this->getTrustedServers();
+		foreach ($trustedServers as $server => $id) {
+			if ($server === $url) {
+				return;
+			}
+		}
+		\PHPUnit\Framework\Assert::fail("Given url $url is not a trusted server");
+	}
+
+	/**
+	 * @Then the trusted server list should include these urls:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function theTrustedServerListShouldIncludeTheseUrls(TableNode $table) {
+		$trustedServers = $this->getTrustedServers();
+		$expected = $table->getColumnsHash();
+
+		foreach ($expected as $server) {
+			$found = false;
+			foreach ($trustedServers as $url => $id) {
+				if ($url === $server['url']) {
+					$found = true;
+					break;
+				}
+			}
+			if (!$found) {
+				\PHPUnit\Framework\Assert::fail("Given url ${server['url']} is not a trusted server");
+			}
+		}
+	}
+
+	/**
+	 * @Given the administrator has added url :url as trusted server
+	 *
+	 * @param string $url
+	 *
+	 * @return void
+	 */
+	public function theAdministratorHasAddedUrlAsTrustedServer($url) {
```

Comment:
This method looks the same as `theAdministratorAddsUrlAsTrustedServerUsingTheTestingApi`
You can write 2 forms of step text and just have 1 method.